internal/node/node.go

```diff
@@ -178,6 +179,15 @@ func NewBus(cfg BusConfig, dir string, walletKey consensus.PrivateKey) (*bus.Bus
 		return nil, nil, err
 	}
 
+	var m *Miner
+	if cfg.Miner {
+		m = NewMiner(cm)
+		if err := cm.ConsensusSetSubscribe(m, ccid, nil); err != nil {
+			return nil, nil, err
+		}
+		tp.TransactionPoolSubscribe(m)
```

Comment:
if you add a `NewBusCustom` where you pass in all components, you would be able to subscribe the miner and keep it off of the `bus` and consolidated to the `TestCluster` right? Wouldn't that be cleaner?

Reply:
I'll wait for @lukechampine to also review this before committing to that. Mostly because I want to get rid of the miner in the bus asap anyway. To replace it with a subscription through the API.